### PR TITLE
Remove repo.alexia.lol

### PR DESCRIPTION
**Pull request by azaz0322**
repo.alexia.lol redirects anyone in the United Kingdom to an (unofficial) site for a petition to stop the E2EE backdoor act from going through in the parliament.
However, this is not useful as most who use this repo are end users who have no idea about this act or petition and wouldnt want to sign it. Additionally, in package managers such as Sileo, the redirect doesn't even show, meaning the repo just doesn't work anymore.
TLDR; Repo doesn't work for anyone in the UK, redirects to bogus petition instead.

### CODE REVIEW

**Comment by tale** [comment]
I'm not inclined to stop indexing a repository because it's broken in the UK. Canister is exclusively hosted in `us-east-1` and will continue to have access to it. I'm sure at some point it'll also be fixed and on top of that all of the packages still show up on Canister for it.

**Comment by ghost** [comment]
It was supposed to be set up in a way that only browser user agents, i.e. not package managers, should be affected by this, so the most it should have broken is depictions, but admittedly I didn't bother testing it properly. I've disabled the redirect for now and purged Cloudflare's cache, hopefully it will go live everywhere soon. (Not sure if I'll even bother re-enabling it.)

Also, next time I'd appreciate a ping about this, at least here in this issue if you can't on Discord due to drama that I won't bring here.
